UI/src/app/components/comfy/comfy.component.ts

```diff
@@ -94,28 +98,31 @@ export class ComfyComponent extends BaseComponent implements OnInit, AfterViewIn
   @ViewChildren(CdkDrag) dragEls: QueryList<CdkDrag>;
 
   drop = new Command({
-    execute: (event: CdkDragDrop<string[]>) => {
+    execute: (event: any) => {
       const { container, previousContainer, previousIndex, currentIndex } = event;
       const data = container.data;
+      const [ area, targetName ] = container.id.split('-');
+      const exists = container.data.find(tableName => previousContainer.data[ previousIndex ] === tableName);
 
       if (previousContainer === container) {
-        if (previousIndex !== currentIndex) {
-          moveItemInArray(data, previousIndex, currentIndex);
+        if (area === Area.Target) {
+          const draggedItemId = event.item.element.nativeElement.id;
+          const mainContainer = this.document.getElementById('main');
```

Comment:
Could we use elementRef instead of document.getElementById here?

Reply:
You've convinced me in your previous PR, that this will work faster

Reply:
I said this about a situation where we have a lot of elements and we should use ViewChildren intsead of ViewChild and we should search for correct 'element. But here I see only one element "main"